farm/modeling/language_model.py

```diff
@@ -1404,8 +1405,13 @@ def load(cls, pretrained_model_name_or_path, language=None, **kwargs):
         else:
             # Pytorch-transformer Style
             dpr_question_encoder.model = transformers.DPRQuestionEncoder(config=transformers.DPRConfig(**kwargs))
-            # load weights from pretrained_model_name_or_path Language model into DPRQuestionEncoder
-            dpr_question_encoder.model.base_model.bert_model = AutoModel.from_pretrained(str(pretrained_model_name_or_path), **kwargs)
+            if pretrained_model_name_or_path == dpr_question_encoder.transformers_modelhub_path:
```

Comment:
Let's better check for if the string "dpr-question_encoder" is in pretrained_model_name_or_path. There might be more models / or local DPR models that we want to load directly...

Similarly for the context encoder below...

Reply:
Fixed it